12_1_Verifying_Your_Tor_Setup.md

```diff
@@ -0,0 +1,18 @@
+# Chapter 12: Verifying Your Tor Setup
+
+In this chapter we will verify tor installation and setup.
+
+```
+~$ sudo -u debian-tor tor --verify-config
+```
+
+If tor is installed correctly you should see an output like this:
+
+```
+Jun 26 21:52:09.230 [notice] Tor 0.4.3.5 running on Linux with Libevent 2.0.21-stable, OpenSSL 1.0.2n, Zlib 1.2.11, Liblzma 5.2.2, and Libzstd N/A.
+Jun 26 21:52:09.230 [notice] Tor can't help you if you use it wrong! Learn how to be safe at https://www.torproject.org/download/download#warning
+Jun 26 21:52:09.230 [notice] Read configuration file "/etc/tor/torrc".
+Configuration was valid
+
```

Comment:
This is great. 

It'd also be helpful to think through what other basic command line help we can give them (that doesn't overlap with the later planned chapters). 

How can they look up what Tor's currently doing? Is there a way to do it from the CLI? From config files? 

What does the display they see mean? (I'm assuming there are just Tor services for the bitcoind port.) 

How do they then access those ports that are now controlled from Tor? From localhost? (I assume it's just normal, but it'd be good to show them why.) From remote?

If there's something *informational*, this would be a good place to put it. If there's something that requires changes, that's in the next chapters.

Chapter 3.1 is a parallel chapter where we mostly looked at a new setup and saw what it meant.

Reply:
I've added two options to test Tor config using bitcoin-cli or querying debug.log.  It will show onion id that will allow connect from external.